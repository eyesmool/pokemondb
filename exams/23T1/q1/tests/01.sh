psql bank -c 'select * from q1 order by suburb;'
