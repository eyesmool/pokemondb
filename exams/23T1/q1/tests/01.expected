      suburb       | ncust 
-------------------+-------
 Bondi Junction    |    13
 Brighton-Le-Sands |    13
 Kirribilli        |    13
(3 rows)
